browser.js

```diff
@@ -1992,4 +1997,4 @@ document.addEventListener('DOMContentLoaded', function() {
 });
 
 }());
-//# sourceMappingURL=browser.js.map
\ No newline at end of file
+//# sourceMappingURL=browser.js.map
```

Comment:
Git kept saying it was missing a newline at the end of the file and it does have a newline :/